src/azure-cli/azure/cli/command_modules/advisor/custom.py

```diff
@@ -132,7 +131,7 @@ def _parse_recommendation_uri(recommendation_uri):
 def _generate_recommendations(client):
     from msrestazure.azure_exceptions import CloudError
 
-    response = sdk_no_wait(True, client.generate)
+    response = client.generate(raw=True)
```

Comment:
@Prasanna-Padmanabhan, 
Here we pass raw=True in method instead of using sdk_no_wait (actually it's not correct to use sdk_no_waitbefore as client.generate is not a long running operation.), please help confirm this change.

Reply:
I don't have the necessary knowledge to sign off on this change. If all existing tests are passing, then you can go ahead.

Reply:
@Prasanna-Padmanabhan, thanks for your confirm. All existing tests are passing, so we'll commit this change.